src/encoded/static/components/genome_browser.js

```diff
@@ -126,6 +126,18 @@ const getDefaultCoordinates = (assembly, annotation, ignoreCache = false) => {
                 file_format: 'variant',
                 path: 'https://encoded-build.s3.amazonaws.com/browser/GRCh38/GRCh38-dbSNP153.vvariants-dir',
             },
+            {
```

Comment:
Do you need to do deduping? In theory, there could be pages where the inserted BigBed appears twice.

Reply:
This is an excellent point, it looks like we're not checking for duplicates anywhere.